index.js

```diff
@@ -101,6 +144,13 @@ async function parseSocketConfig (fileContent) {
     throw new ErrorWithCause('Error when parsing socket.yml config', { cause: err })
   }
 
+  if (!parsedContent?.version) {
```

Comment:
Whats the best way to coerce this to an object that might have a .version property?

Reply:
```suggestion
  if (parsedContent && typeof parsedContent === 'object' && !('version' in parsedContent)) {
```

Three steps:

1. Ensure we have no `null` values (as those match `typeof parsedContent === 'object'`)
2. Ensure its an object
3. Ensure there is no `version` in it

Reply:
I swear I tried that. Oh well, I guess I was slightly off.